packages/@okta/vuepress-site/docs/guides/oie-embedded-common-refresh-tokens/main/nodeexpress/refreshusingthesdk.md

```diff
@@ -0,0 +1,35 @@
+## Refresh using the Javascript SDK
+
+The Javascript SDK has built-in support for auto renewing of access and
+id tokens. This behavior is enabled by default, but can be disabled by
+setting the `tokenManager's` `autoRenew` property to `false`.
+
+```javascript
+const authClient = new OktaAuth({
+    tokenManager: {
+      autoRenew: false
+    }
+  });
+```
+
+With this property set to `false`, you can still manually refresh tokens by
+subscribing to the `tokenManager's` `expired` event and calling it's
```

Comment:
I am not a big fan of writing these in the possessive. I don't think it really needs to be:
 
```suggestion
subscribing to the `tokenManager` `expired` event and calling its
```